e_os.h

```diff
@@ -295,6 +295,64 @@ struct servent *getservbyname(const char *name, const char *proto);
 # endif
 /* end vxworks */
 
+/* ----------------------------- HP NonStop -------------------------------- */
+/* Required to support platform variant without getpid() and pid_t. */
+#ifdef __TANDEM
+#  include <strings.h>
+#  include <netdb.h>
+#  define getservbyname(name,proto)          getservbyname((char*)name,proto)
+#  define gethostbyname(name)                gethostbyname((char*)name)
+#  define ioctlsocket(a,b,c)	ioctl(a,b,c)
+#  ifdef NO_GETPID
+inline int nssgetpid();
+#    ifndef NSSGETPID_MACRO
+#      define NSSGETPID_MACRO
+#      include <cextdecs.h(PROCESSHANDLE_GETMINE_)>
+#      include <cextdecs.h(PROCESSHANDLE_DECOMPOSE_)>
+       inline int nssgetpid()
```

Comment:
This should be `static inline`

Reply:
I will check with the team. Can we fix this later? The person is away.

Reply:
We can fix that later